scheduler/controllers/odigosconfiguration/odigosconfiguration_controller.go

```diff
@@ -57,6 +59,16 @@ func (r *odigosConfigurationController) Reconcile(ctx context.Context, _ ctrl.Re
 		return ctrl.Result{}, err
 	}
 
+	// Read and merge remote config (from central-backend) if it exists
+	// Remote config takes precedence over helm-managed config
+	remoteConfig, err := r.getRemoteConfig(ctx)
+	if err != nil {
+		logger.V(1).Info("No remote config found, using only helm-managed config", "error", err)
```

Comment:
most odigos deployment will not contain the remote config. should we avoid this log line when it's not relevant?